crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -48,6 +48,11 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
+    /// Create a new `Commands` from a queue and an [`Entities`] reference.
```

Comment:
This should warn that if the commands never get applied, they will never get applied.

I would not be surprised if we had some users not recognising this - at least then we can point them to these docs.

What is the expected usage pattern of this anyway?

Reply:
The expected usage is creating multiple `Commands` instances within a `par_for_each` context. I plan on making a wrapper api as well, so most users won't need to use this directly. As for changing the documentation, I'd argue it's not that necessary given the docs on `CommandQueue` should cover it.